src/layer.ts

```diff
@@ -48,12 +52,19 @@ export const runtimeLookup: { [key: string]: RuntimeType } = {
   "python3.7": RuntimeType.PYTHON,
   "python3.8": RuntimeType.PYTHON,
   "python3.9": RuntimeType.PYTHON,
+  "dotnetcore3.1": RuntimeType.DOTNET,
```

Comment:
I think we also want to support the new `dotnet6` runtime as well

Reply:
I don't believe the extension currently supports dotnet6 so might be worth adding there as well

Reply:
dotnet6 doesn't seem to be included in what's getting logged here: ```Serverless: Configuration warning at 'provider.runtime': should be equal to one of the allowed values [dotnetcore2.1, dotnetcore3.1, go1.x, java11, java8, java8.al2, nodejs10.x, nodejs12.x, nodejs14.x, provided, provided.al2, python2.7, python3.6, python3.7, python3.8, ruby2.5, ruby2.7]``` Are we sure it should be added?

Reply:
Liza I think that's a serverless framework error. You may want to bump from v2 to v3. It should have net6.0 runtime support.

Reply:
It looks like this was added to the Serverless framework in v3.4.0:

https://github.com/serverless/serverless/releases/tag/v3.4.0